crates/modelardb_manager/src/remote.rs

```diff
@@ -168,7 +312,115 @@ impl FlightService for FlightServiceHandler {
         let action = request.into_inner();
         info!("Received request to perform action '{}'.", action.r#type);
 
-        if action.r#type == "RegisterNode" {
+        if action.r#type == "InitializeDatabase" {
+            // Extract the list of comma seperated tables that already exist in the server.
+            let server_tables: Vec<&str> = str::from_utf8(&action.body)
+                .map_err(|error| Status::invalid_argument(error.to_string()))?
+                .split(',')
+                .filter(|table| !table.is_empty())
+                .collect();
+
+            // Get the table names in the clusters current database schema.
```

Comment:
I have used `clusters database schema` to mean the normal tables and model tables in the cluster. If we replace with `metadata database` it might be confused with the tables of the metadata database such as `table_metadata`, `model_table_metadata`, and `nodes`.